datasets/bor_inspecties/dataset.json

```diff
@@ -0,0 +1,302 @@
+            "description": "Link naar de gehele schouwronde"
+          },
+          "inspectionRoundName": {
+            "type": "string",
+            "description": "Naam van de schouwronde"
+          },
+          "inspectieRoundPeriodeStart": {
+            "type": "string",
+            "description": "Unix timestamp met geplande startdatum en tijd"
+          },
+          "inspectionRoundPeriodeEnd": {
+            "type": "string",
+            "description": "Unix timestamp met geplande einddatum en tijd"
+          },
+          "inspectionRoundModifiededAt": {
+            "type": "string",
+            "description": "Unix timestamp met laatste modificatiedatum en tijd"
+          },
+          "inspectionIds": {
+            "type": "string",
+            "description": "Het id van de inspectie"
+          },
+          "inspectionCreatedAt": {
+            "type": "string",
+            "description": "Unix timestamp met aanmaakdatum"
+          },
+          "resultsAboveMinvalue": {
+            "type": "boolean",
+            "provenance": "all_results_above_minimum_value",
+            "description": "true ( = geen afkeur) / false ( = bevat afkeur)"
+          },
+          "inspectionFinishedAt": {
+            "type": "string",
+            "description": "Unix timestamp met sluitingsdatum en tijd"
+          },
+          "inspectionResultFiledAt": {
+            "type": "string",
+            "description": "Unix timestamp met laatste modificatiedatum"
+          },
+          "closingUserName": {
+            "type": "string",
+            "description": "Gebruiker die de inspectie heeft gefinaliseerd"
+          },
+          "scoreId": {
+            "type": "string",
+            "description": "Het id van het resultaat"
+          },
+          "score": {
+            "type": "string",
+            "enum": [
+              "A+",
+              "A",
+              "B",
+              "C",
+              "D",
+              "NVT"
+            ],
+            "description": "A+, A, B, C, D of NVT"
+          },
+          "scoreCreatedAt": {
+            "type": "string",
+            "description": "Unix timestamp met aanmaakdatum en tijd."
+          },
+          "creatingUserId": {
+            "type": "string",
+            "description": "Id van de inspecteur. Deze id is uniek ongeacht de evironment."
+          },
+          "creatingUserName": {
+            "type": "string",
+            "description": "Volledige naam van de inspecteur."
+          },
+          "photosPhotoUri": {
+            "type": "string",
+            "description": "Link naar de foto"
+          },
+          "xMeting": {
+            "type": "string",
+            "description": "X-coordinaat van meting (RD EPSG:28992)"
+          },
+          "yMeting": {
+            "type": "string",
+            "description": "Y-coordinaat van meting (RD EPSG:28992)"
+          },
+          "geometry": {
+            "type": "string",
```

Comment:
Het `"type": "string"` moet hier weggehaald worden. Want, de `$ref` legt het type van het veld al vast.

Reply:
geometrie geven we al apart in het veld `geometry` deze kolom is eigenlijk oorspronkelijke kolom waarvan we het geometrie veld afleiden. Dus deze mag nu zo blijven.

Reply:
wordt aangepast.